src/drawing-tool/components/runtime.tsx

```diff
@@ -17,6 +19,14 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
   const readOnly = report || (authoredState.required && interactiveState?.submitted);
   const useSnapshot = authoredState?.backgroundSource === "snapshot";
   const useUpload = authoredState?.backgroundSource === "upload";
+  const authoredBgCorsError = useCorsImageErrorCheck({
+    performTest: authoredState.backgroundSource === "url" && !!authoredState.backgroundImageUrl,
+    imgSrc: authoredState.backgroundImageUrl
+  });
+  if (authoredBgCorsError) {
+    return <div className={css.error}>Authored background image is not CORS enabled, please use a different background
```

Comment:
```suggestion
    return <div className={css.error}>Authored background image is not CORS enabled; please use a different background
```
Either a semicolon or two full sentences to avoid the run-on sentence.

Reply:
Thanks, fixed.